client/box/box-list.view.html

```diff
@@ -1,8 +1,44 @@
+<md-toolbar class="md-hue-1">
+  <div class="md-toolbar-tools">
+    <h3><span>{{box.data.name}}</span></h3>
+    <span flex></span>
+    <md-menu md-position-mode="target-right target">
+      <md-button class="md-icon-button" aria-label="Box settings" ng-click="$mdOpenMenu($event)">
+        <i class="material-icons">more_vert</i>
+      </md-button>
+      <md-menu-content width="4">
+        <md-menu-item ng-show="box.data.owner === main.user.name || main.user.isAdmin">
+          <md-button ng-click="box.edit()" >
+              <div layout="row">
+                <p flex="">Edit</p>
+              </div>
+          </md-button>
+        </md-menu-item>
+        <md-menu-item ng-if="!box.isDeleted" >
```

Comment:
Be sure to hide the delete/undelete buttons if the user isn't the owner or an admin.